src/components/layout/sendTip/SendTip.vue

```diff
@@ -9,7 +9,52 @@
           @click="toggleForm"
         >
       </div>
-      <form @submit.prevent="sendTip">
+      <form
+        v-if="feed === 'posts'"
+        @submit.prevent
+      >
+        <MessageInput
+          v-model="sendPostForm.title"
+          maxlength="280"
+          :placeholder="$t('What are your thougts?')"
+        />
+        <div class="mt-2 d-flex flex-row">
+          <div class="post-actions">
+            <ButtonPlain disabled>
+              <IconPictures />
+            </ButtonPlain>
+            <ButtonPlain disabled>
+              <IconGif />
+            </ButtonPlain>
+            <ButtonPlain disabled>
+              <IconEmoji />
+            </ButtonPlain>
+            <ButtonPlain disabled>
+              <IconPoll />
+            </ButtonPlain>
+            <ButtonPlain disabled>
+              <IconThreeDots />
+            </ButtonPlain>
+          </div>
+          <div class="separator" />
+          <div class="text-right">
+            <ButtonPlain
+              class="btn btn-primary post-submit"
+              type="submit"
+              :disabled="!sendPostForm.title.length || sendingPost"
+              @click="sendPost"
+            >
+              <span class="text-nowrap">
+                <IconPosts /> {{ $t('Post') }}
```

Comment:
Can you apply `text-nowrap` on the button node?
```suggestion
            <ButtonPlain
              class="btn btn-primary post-submit text-nowrap"
              type="submit"
              :disabled="!sendPostForm.title.length || sendingPost"
              @click="sendPost"
            >
              <IconPosts /> {{ $t('Post') }}
```
is `text-nowrap` actually needed?